core/graphelier-service/api/hndlrs/obhandler.go

```diff
@@ -0,0 +1,44 @@
+package hndlrs
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+
+	"graphelier/core/graphelier-service/models"
+
+	"github.com/gorilla/mux"
+)
+
+// JSONOrderbook : Sends an orderbook as json
+func JSONOrderbook(env *Env, w http.ResponseWriter, r *http.Request) error {
+	params := mux.Vars(r)
+	w.WriteHeader(http.StatusOK)
+
+	instrument := params["instrument"]
+	timestamp := params["timestamp"]
+
+	// Convert string to uint64
+	intTimestamp, err := strconv.ParseUint(timestamp, 10, 64)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
should exit function if there is an error here. It'll crash the program if the function continues to execute the following lines if there's an error.

Reply:
would it be better to return err in this case?

Reply:
Function signatures seems to imply it can return an error

Reply:
Ah right